realm/src/main/java/io/realm/internal/SharedGroup.java

```diff
@@ -233,6 +231,18 @@ public boolean compact() {
         return nativeCompact(nativePtr);
     }
 
+
+    /**
+     * Returns the absolute path to the file backing this SharedGroup.
+     *
+     * @return Absolute path to the Realm file.
+     */
+    public String getPath() {
```

Comment:
Since the Java documentation (https://docs.oracle.com/javase/7/docs/api/java/io/File.html) discuss `getPath()` and `getAbsolutePath()` as two different methods, I think this method should be called `getAbsolutePath()`.